src/foam/u2/wizard/data/FacadeWizardletSaver.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.u2.wizard.data',
+  name: 'FacadeWizardletSaver',
+  extends: 'foam.u2.wizard.data.ProxySaver',
+
+  imports: [
+    'wizardlets'
+  ],
+
+  properties: [
+    {
+      class: 'FObjectArray',
+      name: 'facadeWizardletSpecs',
+      of: 'foam.u2.wizard.wizardlet.FacadeWizardletSpec',
+      factory: function(){
```

Comment:
Isn't this already the default factory: for FObjectArrays?

Reply:
Removed.